multiversx_sdk/network_providers/config.py

```diff
@@ -1,13 +1,23 @@
+from dataclasses import dataclass, field
 from typing import Any, Optional
 
 
+@dataclass
+class RequestsRetryOptions:
+    retries: int = 3
+    backoff_factor: float = 0.05
+    status_forecelist: list[int] = field(default_factory=lambda: [500, 502, 503, 504])
```

Comment:
They seem fine. If there are constants in Python standard library, we can use them.